### DISCUSSION TITLE
RS485 comms - change the controller address?

### DISCUSSION THREAD

**Discussion by BrokenPork** [tagyoureit/nodejs-poolController#952] [Nixie]
I am fairly new to this app, but it looks like everything i need. Prob will migrate all my other controls when i rebuild my panel. I got a new VS pump with rs485. I was able to test and comm with it outside of this project. When i set it up here, I am not getting good communication. I have found the issue but can't seem where to change it. 

The  source and destination addresses i use are 0x60 for the pump and 0x10 for the controller. This system is sending out the the controller as 0x21 for the address. Looks like it is trying to emulate some other controller. 

Where do I update the controller address?

How I troubleshot my issue to come up with my conclusion:
-Have used the message manager and changed the controller address to 0x10 and i got successful writes.
-turned poolcontroller app off and used my terminal software on the rs485 port with the 0x21 address and confirmed the pump doesn't like that there either. 

thanks

**Comment by tagyoureit** [comment]
We have never had anyone need to change the addresses of this app.  The address we use isn't used by other equipment.  Something else is going on if the communications are not "good".  

The address of the pump can be changed in the setup menu for the pump.
